src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoOffhand.kt

```diff
@@ -100,7 +101,7 @@ internal object AutoOffhand : Module(
 
             transactionLog[it.packet.actionNumber] = it.packet.wasAccepted()
             if (!transactionLog.containsValue(false)) {
-                movingTimer.reset(-175L) // If all the click packets were accepted then we reset the timer for next moving
+                movingTimer.reset(-max(0, delay - 1).toLong()) // If all the click packets were accepted then we reset the timer for next moving
```

Comment:
Is it intentional to cap resetting movingTimer to 0?

Reply:
delay can be 0 which means, 0 - 1 is -1. -(-1) means it would add check for 1 tick or something?

I may be wrong here with the logic and will update it. I've been off meds for 48 hours and just got them so I probably made monkey code. Xiaro might want to look at it or whomever implement the AutoOffhand for the transactionLog original.

Reply:
That would indeed be @Xiaro

Reply:
Reset with negative value will make the timer reset to an earlier time in the tick timer.

Reply:
And the moving timer wasn't for limiting the moving speed. It was for waiting for responses from server with a timeout

Reply:
> And the moving timer wasn't for limiting the moving speed. It was for waiting for responses from server with a timeout

Is there a better way to accomplish this possibly? or should we keep it at a fixed 3 ticks? Just curious if there is something we should consider here. Chain popping would sometimes not move a totem fast enough for me.

Reply:
> > And the moving timer wasn't for limiting the moving speed. It was for waiting for responses from server with a timeout
> 
> Is there a better way to accomplish this possibly? or should we keep it at a fixed 3 ticks? Just curious if there is something we should consider here. Chain popping would sometimes not move a totem fast enough for me.

Separate the setting into Confirm Timeout and Delay

Reply:
Yeah I'll put that out there in a few minutes once I get around to it after real life chores.